frontend/src/pages/planner/CurriculumSelector.tsx

```diff
@@ -26,9 +28,10 @@ const Selector = memo(function _Selector ({
   data,
   value,
   canDeselect,
+  showCode,
   onChange
 }: SelectorProps): JSX.Element {
-  const selectedOption = value !== undefined && value !== null ? (data[value] ?? { name: 'Minor desconocido', code: value }) : { name: 'Por Seleccionar', code: null }
+  const selectedOption = value != null ? (data[value] ?? { name: `${name} Desconocido` }) : { name: 'Por Seleccionar' }
   if (value !== undefined && value !== null && data[value] === undefined) {
     useEffect(() => {
       toast.warn(`Tu ${name} todavía no está soportado oficialmente. Los cursos pueden estar incorrectos, revisa dos veces.`, {
```

Comment:
El uso de minúscula en `name` intencional, por este toast. Creo que esto debiese arreglarlo:

```suggestion
      toast.warn(`Tu ${name.toLowerCase()} todavía no está soportado oficialmente. Los cursos pueden estar incorrectos, revisa dos veces.`, {
```

Reply:
Oh, gracias. Si cache que era minuscula, pero se necesitaba con mayuscula mas arriba y era super feo el `name[0].to_upper() + name[1:]`